README.md

```diff
@@ -1 +1,72 @@
 # babel-plugin-transform-react-pug
+
+The official means by which you can use pug in your react components, replaces the use of react-jade when moving to pug.
+
+This plugin transforms the pug inside of your react components.
+
+## Installation
+```
+npm install babel-plugin-transform-react-pug
+```
+## Usage
+.babelrc
+```js
+{
+  "presets": ["forbeslindesay"],
+  "plugins": [
+    "transform-react-pug",
+  ]
+}
+```
+## Examples
+
+### Example 1
+You can now create a react component with your pug inside it.
+```js
```

Comment:
nit, please add newlines between headings and paragraphs and between paragraphs and blocks of code.